aspnetcore/security/docker-compose-https.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Hosting ASP.NET Core image in container using docker compose with HTTPS
+author: ravipal
+description: Learn how to host ASP.NET Core Images with Docker Compose over HTTPS
+monikerRange: '>= aspnetcore-2.1'
+ms.author: ravipal
+ms.custom: mvc
+ms.date: 07/05/2019
+no-loc: ["Let's Encrypt"]
+uid: security/docker-compose-https
+---
+# Hosting ASP.NET Core images with Docker Compose over HTTPS
+
+
+ASP.NET Core uses [HTTPS by default](/aspnet/core/security/enforcing-ssl). [HTTPS](https://en.wikipedia.org/wiki/HTTPS) relies on [certificates](https://en.wikipedia.org/wiki/Public_key_certificate) for trust, identity, and encryption.
+
+This document explains how to run pre-built container images with HTTPS.
+
+See [Developing ASP.NET Core Applications with Docker over HTTPS](https://github.com/dotnet/dotnet-docker/blob/master/samples/run-aspnetcore-https-development.md) for development scenarios.
+
+This sample requires [Docker 17.06](https://docs.docker.com/release-notes/docker-ce) or later of the [Docker client](https://www.docker.com/products/docker).
+
+## Prerequisites
+
+The [.NET Core 2.2 SDK](https://dotnet.microsoft.com/download) or later is required for some of the instructions in this document.
+
+## Certificates
+
+A certificate from a [certificate authority](https://wikipedia.org/wiki/Certificate_authority) is required for [production hosting](https://blogs.msdn.microsoft.com/webdev/2017/11/29/configuring-https-in-asp-net-core-across-different-platforms/) for a domain. [Let's Encrypt](https://letsencrypt.org/) is a certificate authority that offers free certificates.
+
+This document uses [self-signed development certificates](https://en.wikipedia.org/wiki/Self-signed_certificate) for hosting pre-built images over `localhost`. The instructions are similar to using production certificates.
+
+For production certs:
+
+* The `dotnet dev-certs` tool is not required.
+* Certificates do not need to be stored in the location used in the instructions. Any location should work, although storing certs within your site directory is not recommended.
```

Comment:
```suggestion
* Certificates don't need to be stored in the location used in the instructions. Any location should work, although storing certificates within your site directory is not recommended.
```

Reply:
> Any location should work, although storing certificates within your site directory is not recommended.

This is saying two things. How about something like

Store the certificates in any location outside the site directory.